src/ipc.js

```diff
@@ -68,10 +83,18 @@ var fluid = fluid || require("infusion");
 
         listeners: {
             onCreate: [
+                "{that}.start()"
```

Comment:
Seems puzzling that onCreate operates "start" but onDestroy does not operate "stop"?